src/Rules/DbBoolean.php

```diff
@@ -35,4 +34,11 @@ public function message(): string
 
         return $validator->errors()->first($this->attribute);
     }
+
+    public function validate(string $attribute, mixed $value, Closure $fail): void
```

Comment:
Please re-implement this in a cleaner way in order to avoir passes and message methods